software/chromium.md

```diff
@@ -0,0 +1,35 @@
+Chromium browser on AWS Graviton
```

Comment:
Are these special to Graviton or to AL2023?  If there is an issue getting Chromium on Graviton perhaps its better to add those directions to the Chromium project?

Reply:
Basically, the issue is that none of the major distros (including AL2/AL2023) have convenient packages available for Chromium for the arm64 architecture. It's not really a Chromium issue, since it can be built from source. This purpose of this page is to help customers install it without building it from source themselves.

Reply:
Would it be better in the longer term that those Linux distros have the packages available for installation on aarch64? Looking through the installation instructions, it has very specific version dependencies, so probably need to be explicitly maintained in the future.

Reply:
@yd2102 Of course it would be better, but in the meantime, customers would like to use Graviton for automated web testing today. We can continue to push on Linux distros to include prebuilt Chromium packages for arm64 while providing workarounds here. This repo is a living document and so I expect it to evolve as the software ecosystem does.

Reply:
I need chrome for [Puppeteer](https://github.com/puppeteer/puppeteer) which is a NodeJS framework to interface to websites via (headless) chrome.